docs/VM_Guide.md

```diff
@@ -33,6 +33,13 @@ The most important method is ::evmc_instance::execute() because it executes EVM
 Remember that the Host is allowed to invoke the execute method concurrently
 so do not store data related to a particular execution context in the VM instance.
 
+Before a client can actually execute a VM, it is important to implement the three
+basic methods for querying name (::evmc_instance::name), version (::evmc_instance::version)
```

Comment:
`name` and `version` are not really methods.

Reply:
Fixed.